src/types/contracts/moneyMarket/model.ts

```diff
@@ -150,6 +150,20 @@ type ParsedUserPositionResponse = {
   loanLiquidationPoint: string,
 }
 
+// New types for public events
+type PublicEvent = {
+  timestamp: number,
+  action: Record<string, any>
+}
+
+type PaginatedPublicEvent = {
```

Comment:
use ParsedPagination<PublicEvent[]>

Reply:
So the reason i did not use this name is becasue the name already exists.

Reply:
Which name already exists?